pkgs/applications/virtualization/quickgui/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quickgui";
+  version = "1.2.8";
+
+  src = fetchurl {
+    url = "https://github.com/quickemu-project/quickgui/releases/download/v${version}/quickgui_${version}-1_lunar1.0_amd64.deb";
+    sha256 = "sha256-crnV7OWH5UbkMM/TxTIOlXmvqBgjFmQG7RxameMOjH0=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+  ];
+
+  buildInputs = [
+    wrapGAppsHook
+  ];
+
+  unpackCmd = "dpkg-deb -x $curSrc source";
+
+  installPhase = ''
+    runHook preInstall
+
+    mv usr $out
+
+    substituteInPlace $out/share/applications/quickgui.desktop \
+      --replace "/usr" $out
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A Flutter frontend for quickemu";
+    homepage = "https://github.com/quickemu-project/quickgui";
+    downloadPage = "https://github.com/quickemu-project/quickgui/releases/tag/v${version}";
+    changelog = "https://github.com/quickemu-project/quickgui/compare/v1.2.7...v1.2.8";
+    maintainers = [ maintainers.heyimnova ];
+    mainProgram = "quickgui";
+    platforms = platforms.linux;
+    sourceProvenance = [ sourceTypes.binaryNativeCode ];
```

Comment:
```suggestion
    maintainers = [ lib.maintainers.heyimnova ];
    platforms = lib.platforms.linux;
    sourceProvenance = [ lib.sourceTypes.binaryNativeCode ];
    mainProgram = "quickgui";
```

Reply:
PS.: is it too hard to compile the source?

Reply:
I did try to use mkFlutterApp but it didn't work and there seems to be no documentation for building Flutter applications with Nix, I did find two threads in the NixOS forums however neither of them could work out what to do in their own cases.

Reply:
Good to know. I will backlog this on issues.